modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -24,41 +26,257 @@ Open Source users: please report issues in the https://github.com/tinymce/tinymc
+
+.Example
+[source, js]
+----
+tinymce.init({
+  selector: "textarea",
+	sandbox_iframes: false,
+});
+----
+
+This follows our policy that we ship the most secure version of {productname} possible, while giving our customers the opportunity to make informed decisions on dialing back protections to a level that fits their use-case.
+
+[[convert-unsafe-embeds-option]]
+=== `convert_unsafe_embeds`
+
+A previous Security penetration test surfaced a security issue with {productname} regarding object tags, which was a legacy method of inserting an external resource, such as a video, a PDF.
+
+{productname} has determined that the object tag is regarded as unsafe, as it may execute javascript inside the {productname} content.
+
+{productname} has a configuration option that converts object tags to modern equivalents such as `<img>, <video>` and `<audio>` tags, such as;
+
+.Example of before/after conversion.
+[source, html]
+----
+<!-- Before Conversion -->
+<object type="video/mp4" data="https://sneak-preview.tiny.cloud/3adc27b5-bb2f-49f0-9ccc-72b7c48313b0/bad.mov"></object>
+
+<!-- After Conversion -->
+<video src="https://sneak-preview.tiny.cloud/3adc27b5-bb2f-49f0-9ccc-72b7c48313b0/bad.mov" controls="controls"></video>
+----
+
+In {productname} 7.0 by default;
+
+* all object tags will be converted by default by loading the editor.
+* the `convert_unsafe_embeds` option default value will change from `false` to `true` making it opt out instead of opt in.
+
+[NOTE]
+To prevent automatic conversion of content, customers are advised to add `convert_unsafe_embeds: false` to their {productname} init configuration.
+
+[[highlight-on-focus]]
+=== `highlight_on_focus`
+
+Previously, the default value for the editor configuration option `highlight_on_focus` was set to `false` by default. In {productname} 7.0 this option is now by default set to `true`
+
+As a result, the focus outline for the editor will by default be displayed when the focus is on the editor.
+
+Customers who have created their own custom skins, or have implemented their own outline and wish to not to use the default one will need to add the `highlight_on_focus: false` option to their {productname} configuration.
+
+[NOTE]
+This change does not impact editors using inline mode.
+
+[[force-hex-color]]
+=== `force_hex_color`
+
+Previously in {productname} 6, all colors in the content HTML where set to use `RGB` values by default. As the common practice is using `hex` values this change will be **reverted**.
+
+.Example
+[source, html]
+----
+<!-- example using RGB values -->
+<p>Hello <span style="color: rgb(255, 0, 0);">red</span> color</p>
+
+<!-- example using HEX values -->
+<p>Hello <span style="color: #ff0000">red</span> color</p>
+----
+
+In {productname} 7.0, the default will be reverted back to using `hex` values by default.
+
+[IMPORTANT]
+The option `force_hex_color` will be deprecated and removed in a later version of {productname}.
```

Comment:
@danoaky-tiny Can you provide any specific info/changes to this section once we have more information on this option being removed.

Reply:
```suggestion
[[force-hex-color]]
=== `force_hex_color`

Previously in {productname} 6, all colors in the content HTML where set to use `RGB` values by default. As the common practice is using `hex` values this change will be **reverted**.

.Example
[source, html]
----
<!-- example using RGB values -->
<p>Hello <span style="color: rgb(255, 0, 0);">red</span> color</p>

<!-- example using HEX values -->
<p>Hello <span style="color: #ff0000">red</span> color</p>
----

In {productname} 7.0, the default will be reverted back to using `hex` values by default.

[IMPORTANT]
The option `force_hex_color` will be removed altogether in {productname} 7.0.
```

Or merge that `[IMPORTANT]` line with the one above it maybe